dev-requirements-py3.txt

```diff
@@ -14,4 +14,4 @@ beautifulsoup4==4.8.1
 pypdf2==1.26.0
 freezegun==0.3.12
 vulture>=1.4
-git+https://github.com/coyo8/parinx.git@6493798ceba8089345d970f71be4a896eb6b081d
```

Comment:
plase cenge to 1.0.7

Reply:
it's at line 4
the file wasn't really changed as you can see but I'll revert it anyway